sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb

```diff
@@ -287,7 +287,8 @@
     "featurestore_name = \"my-featurestore\"\n",
     "featurestore_location = \"eastus\"\n",
     "featurestore_subscription_id = os.environ[\"AZUREML_ARM_SUBSCRIPTION\"]\n",
-    "featurestore_resource_group_name = os.environ[\"AZUREML_ARM_RESOURCEGROUP\"]"
+    "featurestore_resource_group_name = os.environ[\"AZUREML_ARM_RESOURCEGROUP\"]\n",
+    "feature_version = \"<FEATURE_VERSION>\""
```

Comment:
what is feature_version?

Reply:
Is it not visible to user?

Reply:
all values with "<>" are supposed to be replaced with user defined values when user execute the notebooks.

Reply:
Can you rename the constant FEATURE_VERSION to VERSION? As it is used for featureset and also for featurestore entity.

Reply:
make sense. I'll make the change